src/03-deals/deals.apib

```diff
@@ -22,11 +22,13 @@ Get a list of deals.
                 + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
             + phase_id: `1758e847-cc1f-04ce-bd17-5a852e8c01b4` (string, optional)
             + estimated_closing_date: `2018-02-11` (string, optional, nullable)
+            + estimated_closing_date_before: `2020-01-31` (string, optional) - This date is inclusive
+            + estimated_closing_date_after: `2020-01-01` (string, optional) - This date is inclusive
```

Comment:
Q: same question here.

Reply:
We checked other endpoints and discussed with Stijn

Reply:
What is the reasoning to go for inclusive datetime properties?

To me this feels like adapting the API docs to the way it is currently implemented. This further introduces inconsistencies across the API plus the property name does not reflect the way it behaves (hence the need to add the comments to state it's inclusive).

Reply:
B: As mentioned, let's go for the suffixes _from and _until to make it more explicit the dates are inclusive rather than exclusive. Then we can also remove the comment on the property.